src/EnergyPlus/UnitarySystem.cc

```diff
@@ -18034,6 +18262,36 @@ namespace UnitarySystems {
                                         OutputProcessor::SOVTimeStepType::System,
                                         OutputProcessor::SOVStoreType::Average,
                                         state.dataUnitarySystems->unitarySys[sysNum].Name);
+                    if (((state.dataUnitarySystems->unitarySys[sysNum].m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPSimple ||
+                          state.dataUnitarySystems->unitarySys[sysNum].m_CoolingCoilType_Num ==
+                              DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit) &&
+                         state.dataUnitarySystems->unitarySys[sysNum].m_NumOfSpeedCooling > 1) ||
+                        ((state.dataUnitarySystems->unitarySys[sysNum].m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingWaterToAirHPSimple ||
+                          state.dataUnitarySystems->unitarySys[sysNum].m_HeatingCoilType_Num ==
+                              DataHVACGlobals::Coil_HeatingWaterToAirHPVSEquationFit) &&
+                         state.dataUnitarySystems->unitarySys[sysNum].m_NumOfSpeedHeating > 1)) {
```

Comment:
Can't you simply use:

    if (this->m_MultiSpeedCoolingCoil || this->m_MultiSpeedHeatingCoil)

Reply:
@rraustad It may not be simplified, because these output variables are used only for Coil_Cooling/heatingWaterToAirHPSimple and Coil_Cooling/heatingWaterToAirHPVSEquationFit object. m_MultiSpeedCoolingCoil and m_MultiSpeedHeatingCoil many cover other air source coils also.